src/redisearch_rs/inverted_index/src/lib.rs

```diff
@@ -34,6 +34,17 @@ pub mod raw_doc_ids_only;
 #[doc(hidden)]
 pub mod test_utils;
 
+// Manually define some C functions, because we'll create a circular dependency if we use the FFI
+// crate to make them automatically.
+unsafe extern "C" {
+    /// Checks if a value (distance) is within the given geo filter.
+    ///
+    /// # Safety
+    /// The [`GeoFilter`] should not be null and a valid instance
```

Comment:
Could we use `NonNull` then?

Reply:
I just need something simple to match this

https://github.com/RediSearch/RediSearch/blob/aa1241c5da5c08be956f8bdc90c5b2a4f348d0be/src/geo_index.h#L88

Because adding `geo_index.h` to the ffi crate opens a whole new can of worms